addons/website/static/src/js/widgets/theme.js

```diff
@@ -743,6 +746,20 @@ var ThemeCustomizeDialog = Dialog.extend({
             $dropdown.prepend($btn);
         });
     },
+    /**
+     * @private
+     */
+    _addGoogleFont: function (font) {
+        var values = {};
+        var googleFontsProperty = this.style.getPropertyValue('--google-fonts').replace(/\s*,\s*/g, "','").trim();
+        if (googleFontsProperty) {
+            values['google-fonts'] = "('" + googleFontsProperty + "','" + font + "')";
```

Comment:
I'm afraid I don't see the problem. Yes, it is working on Chrome + Firefox. For example, here is what I get like output when I add "Lato" font: `('Roboto Slab','Poppins','PT Sans Narrow','Lato')`

Reply:
What does the `googleFontsProperty` contains when there are those fonts in it ?

Reply:
For example If I add again a new font, `googleFontsProperty` = `Roboto Slab','Poppins','PT Sans Narrow','Lato`

Reply:
Ok, I didin't see you replaced the `,` by `','`... not very clear :/ Do we really need the quotes and parenthesis ? Can't you just do:
```javascript
var values = {
    'google-fonts': this.style.getPropertyValue('--google-fonts') + ', ' + font,
};
```
?

Reply:
Mmh forget that, maybe I am missing something here, I will check

Reply:
```
var values = {
    'google-fonts': this.style.getPropertyValue('--google-fonts') + ', ' + font,
};
```

I don't see how this could work this way.
Ok, thanks ;)

Reply:
Ok, add a space after each `,` so that it is clearer when debugging with the SCSS editor and it is fine.